InvenTree/InvenTree/views.py

```diff
@@ -637,7 +637,8 @@ def get_context_data(self, **kwargs):
             ctx["rates_updated"] = None
 
         # load locale stats
-        STAT_FILE = settings.BASE_DIR.joinpath('InvenTree/locale_stats.json').abolute()
+        STAT_FILE = settings.BASE_DIR.joinpath('InvenTree/locale_stats.json').absolute()
```

Comment:
Comment: Interesting that flake8 is not catching this. Maybe we should add pylint to the pre-commit stack?

Reply:
Maybe because it's a method name and it can't fully guarantee that `abolute` method does not exist?

Reply:
I will look into type checking for this.